spec/services/create_account_verified_event_spec.rb

```diff
@@ -1,14 +1,13 @@
 require 'rails_helper'
 
 describe CreateVerifiedAccountEvent do
-  let(:password) { 'password' }
   let(:user) do
-    create(:user, :signed_up, password: password) do |user|
+    create(:user, :signed_up) do |user|
```

Comment:
The user doesn't need to be `:signed up` either. This will avoid generating a recovery code, which will speed up the test.